tests/test_grids.py

```diff
@@ -255,7 +255,7 @@ def bath_func(lon):
     for i in range(depth_g0.shape[0]):
         for k in range(depth_g0.shape[2]):
             depth_g0[i, :, k] = bath[i] * k / (depth_g0.shape[2] - 1)
-    depth_g0 = depth_g0.transpose()  # we don't change it on purpose, to check if the transpose op if fixed in jit
+    depth_g0 = depth_g0.transpose()
```

Comment:
Briefly looking at this I'm not sure what this line of code does, or what the comment is about. Any idea? Perhaps we can improve the comment?